Resources/ServerInfo/_NF/Guidebook/Shipyard/Lantern.xml

```diff
@@ -66,9 +49,9 @@
   <GuideEntityEmbed Entity="SheetPlasma"/>
   </Box>
   
-  - Check if the P.A.C.M.A.N. generator unit is anchored to the floor.
-  - Check if the P.A.C.M.A.N. generator unit has fuel. For extended flights make sure that you have enough fuel stockpiled to sustain prolonged power generation during flight.
-  - Check if the P.A.C.M.A.N. generator unit is set to HV output.
+  - Check that the P.A.C.M.A.N. generator unit is anchored to the floor.
+  - Check that the P.A.C.M.A.N. generator unit has fuel. For extended flights make sure that you have enough fuel stockpiled to sustain prolonged power generation during flight.
+  - Check that the P.A.C.M.A.N. generator unit is set to HV output.
   - Set Target Power for 10-15** [bold]k[/bold]W.
```

Comment:
This range is oddly large.
```suggestion
  - Set Target Power for 10-15** [bold]kW[/bold].
```

Reply:
True, will have to spawn it in and see what the actual range is

Reply:
Turns out the lantern uses solars to supplement the power requirements. 22.6kW of total load without the SMES charging. 

Depending on the orientation of the sun relative to the ship the solars make up to 19kW.

on average the pacman fluctuates between 10-12kW, but can rarely max out. If you wanted to optimize fuel use you could run it at 10 but there'd be brown outs.